packages/core/helper-plugin/src/components/Form.tsx

```diff
@@ -2,23 +2,23 @@ import React, { useEffect, useRef } from 'react';
 
 import { Form as FormikForm, useFormikContext } from 'formik';
 
-const Form = (props) => {
-  const formRef = useRef(null);
+const Form = ({ ...props }) => {
+  const formRef = useRef<HTMLFormElement>(null);
```

Comment:
we know `formRef` will be defined so we can do:

```suggestion
  const formRef = useRef<HTMLFormElement>(null!);
```

Which will remove your unnecessary optional chaining

Reply:
Thanks a lot @joshuaellis